utils/https.c

```diff
@@ -21,22 +21,43 @@ status_t send_https_msg(char const *host, char const *port, char const *api, con
   status_t ret = SC_OK;
 
   set_post_request(api, host, atoi(port), msg, &req);
-  http_parser_settings settings;
+  http_parser_settings settings = {};
   settings.on_body = parser_body_callback;
 
   connect_info_t info = {.https = true};
-  /* FIXME:Provide some checks here */
-  http_open(&info, ssl_seed, host, port);
-  http_send_request(&info, req);
-  http_read_response(&info, res, sizeof(res) / sizeof(char));
-  http_close(&info);
+
+  ret = http_open(&info, ssl_seed, host, port);
+  if (ret != SC_OK) {
+    // FIXME: Change to default logger
+    printf("http(s): http open error\n");
```

Comment:
Use logger when possible.

Reply:
The issue #578 will resolve this. Should we replace the printf to default logger here?

Reply:
Do it here. Don't use printf for error messages.